infra/base-images/base-builder/srcmap

```diff
@@ -25,8 +25,14 @@ function jq_inplace() {
   F=$(tempfile) && cat $1 | jq "$2" > $F && mv $F $1
 }
 
+PATHS_TO_SCAN="$SRC"
+
+if [[ $FUZZING_LANGUAGE == "go" ]]; then
+  PATHS_TO_SCAN="$PATHS_TO_SCAN $GOPATH"
```

Comment:
What's in $GOPATH if this isn't a Go project? Can we use that to determining whether or not to scan it, rather than introducing another env var?

Reply:
It's always set to `/root/go` in the base-builder and always has a few packages we install, so scanning unconditionally will result in irrelevant Go packages revisions in srcmap of any project we build